docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+
+- Handler type: either Signal or Query
+- Handler name
+- Workflow function that handles the Signal or Query
+
+This can be useful for debugging or introspecting which Signal or Query a Workflow currently supports.
+
+## Built-in Temporal CLI dev server runner
+
+| Feature                                                                             | Go  | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------------------------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Built-in Temporal CLI dev server runner](#built-in-temporal-cli-dev-server-runner) | Yes | No   | Yes        | Yes    | Yes  | No  |
+
+This allows starting the CLI dev server from code rather than manually from the terminal.
+It handles starting and stopping the server automatically around your Workflows.
+
+## Bulk replayer
+
+| Feature                         | Go  | Java | TypeScript | Python | .Net | PHP |
+| ------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Bulk replayer](#bulk-replayer) | Yes | Yes  | Yes        | Yes    | Yes  | No  |
+
+The bulk replayer enables efficient parallel replay of many Workflow histories from a JSON file or an array of Workflow history event objects for testing and debugging Workflows in Temporal.
+
+## Cancellation
+
+| Feature                       | Go  | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Cancellation](#cancellation) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+Cancellation in Temporal SDKs refers to the ability to stop the execution of a Workflow, Activity, or other operation before it completes normally.
+
+Each SDK provides APIs and abstractions to enable robust cancellation handling in Workflows and Activities.
+
+## Child Workflows
+
+All SDKs support the ability to spawn Child Workflows.
+
+| Feature                                 | Go  | Java | TypeScript | Python | .Net | PHP |
+| --------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Child Workflow APIs](#child-workflows) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+A Child Workflow Execution is a Workflow Execution that is spawned from within another Workflow.
+
+Child Workflows in Temporal SDKs refer to Workflows that are started from within another Workflow.
+
+Each SDK provides APIs and abstractions to handle the complexity behind the scenes.
+
+## Continue-As-New
+
+| Feature                             | Go  | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Continue as New](#continue-as-new) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+Continue-As-New is the mechanism by which all relevant state is passed to a new Workflow Execution with a fresh Event History.
+
+## Delay Start Workflow
```

Comment:
This is not deserving of a top-level feature any more than every other workflow start option.